src/Methods/PayUponPickupPaymentMethod.php

```diff
@@ -72,14 +73,14 @@ public function getName(string $lang = 'de'): string
     }
 
     /**
-    * Get PayUponPickup Fee
+    * Return Payment Method Fee
     *
     * @return float
     */
     public function getFee(): float
     {
 
-        return 0.00;
+        return -10.00;
```

Comment:
Why this fixed value of -10? And why is there code after the return?

Reply:
Oh this was just a mistake. I used this value to test an other case 😄 
Fixed now.

Reply:
Ok, but still strange that there is code after the return.

Reply:
Year thats right. I think we still left the code there as an example.